lib/internal/backend/repository/repository2.go

```diff
@@ -512,7 +504,6 @@ func (rb *RepositoryBackend) getLayerV2(layerID string, dockerURL *types.ParsedD
 			if err != nil {
 				return nil, nil, err
 			}
-			defer res.Body.Close()
```

Comment:
I just noticed that the [original res](https://github.com/appc/docker2aci/pull/180/files#diff-acab445bd3e69965398509002dd14a18L499) doesn't get closed if this if block gets dropped into. Would you mind adding a `res.Body.Close()` around like 507 while you're here?